src/ert/_c_wrappers/fm/ecl/ecl_run.py

```diff
@@ -307,6 +307,8 @@ def execEclipse(self, eclrun_config=None):
             )
             env = eclrun_config.run_env if use_eclrun else self._get_legacy_run_env()
 
+            # pylint: disable=consider-using-with
```

Comment:
Why we don't need `, encoding="utf-8"` in the `with ... open...` above?

Reply:
On line 297, the file is opened in binary mode, and then it does not make sense to specify the encoding.